src/Hooks/usePaintingSummary.js

```diff
@@ -0,0 +1,21 @@
+import {useState, useEffect} from 'react';
+
+const usePaintingSummary = (contentId) => {
```

Comment:
Beautiful custom hook

Reply:
The page looks great. I like getting rid of the back button. It looks very clean this way.